app/controllers/dashboard_controller.rb

```diff
@@ -12,8 +12,10 @@ def index
   end
 
   def create
```

Comment:
Doesn't appear so, since the latest rewrite.  I'd guess what you did was perhaps the intention, but was left incomplete.